File: app/src/main/java/org/techtown/find_gas_station/Data/kakaoResponseModel/oilAvg/RESULT.java
package org.techtown.find_gas_station.Data.kakaoResponseModel.oilAvg;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RESULT {

    @SerializedName("OIL")
    @Expose
    private OIL[] oil;

    public OIL[] getOil(){return oil;}

    public void setOil(OIL[] oil){this.oil = oil; }

    @Override
    public String toString(){
        return "ClassPojo [OIL = "+ oil+"]";
    }
}

File: app/src/main/java/org/techtown/find_gas_station/Data/kakaoResponseModel/oilAvg/Result.kt
package org.techtown.find_gas_station.Data.kakaoResponseModel.oilAvg

import com.google.gson.annotations.SerializedName

data class Result(

    @SerializedName("OIL")
    val oilAveragePriceInfo : List<OilAveragePriceInfo>
)
